README.md

```diff
@@ -1149,6 +1155,9 @@ IEEE Robotics and Automation Letters (RA-L) 2024. [PDF](https://arxiv.org/pdf/24
 *[Deep Polarization Reconstruction with PDAVIS Events](https://openaccess.thecvf.com/content/CVPR2023/html/Mei_Deep_Polarization_Reconstruction_With_PDAVIS_Events_CVPR_2023_paper.html)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [PDF](https://openaccess.thecvf.com/content/CVPR2023/papers/Mei_Deep_Polarization_Reconstruction_With_PDAVIS_Events_CVPR_2023_paper.pdf), [code](https://github.com/SensorsINI/e2p).
 
+- [Muglikar et al., CVPR 2023](#Muglikar23CVPR),  
```

Comment:
I am not sure why it appears that you modified the previous entry
![image](https://github.com/user-attachments/assets/d1f5f852-e639-46e9-8ea3-a153e89bebf3)

Did you add an extra space and it looks like a new list, rather than a new bullet point?

Reply:
Please always check the preview on github

Reply:
Not sure why it appears like that or how to fix it.
But I updated the polarization index that it refers correctly to this section from the main list in the recent comment, maybe that caused this?

Reply:
There was an extra space... (gave you the hint)... it is fixed now.